example/filename.dg

```diff
@@ -1 +1 @@
-digraph G {rankdir="LR";node[shape="box"];  Fragment -> FragmentRefine[label="refine_que : queue"]; FragmentRefine -> Deduplicate[label="deduplicate_que : queue"] Deduplicate -> Compress[label="compress_que : queue"] Deduplicate -> Reorder[label="reorder_que : queue"] Reorder -> Compress[label="reorder_que : queue"] }
\ No newline at end of file
+digraph G {rankdir="LR";node[shape="box"];  Fragment -> FragmentRefine[label="refine_que : queue"]; FragmentRefine -> Deduplicate[label="deduplicate_que : queue"] Deduplicate -> Compress[label="compress_que : queue"] Deduplicate -> Reorder[label="reorder_que : queue"] Reorder -> Compress[label="reorder_que : queue"] }
```

Comment:
Please have the dg file include only the actual information that needs to be supported. Please remove info such as the shape of the node or what is "rankdir".